src/hotspot/share/interpreter/linkResolver.cpp

```diff
@@ -323,6 +323,9 @@ void LinkResolver::check_klass_accessibility(Klass* ref_klass, Klass* sel_klass,
     char* msg = Reflection::verify_class_access_msg(ref_klass,
                                                     InstanceKlass::cast(base_klass),
                                                     vca_result);
+
+    // Names are all known to be < 64k so we know this formatted message is not excessively large.
+
```

Comment:
Can you move this comment to before the first fthrow call at 331?  The other fthrow has a msg so doesn't really apply and the comment looks better as just one line like the other places.

Reply:
The comment still applies to the msg created by:
```
char* msg = Reflection::verify_class_access_msg(ref_klass,
                                                    InstanceKlass::cast(base_klass),
                                                    vca_result);
```
which is also known to be limited by class name symbol lengths. That is why I placed the comment prior to both fthrow calls.